packages/esm-patient-forms-app/src/forms/form-entry.component.tsx

```diff
@@ -66,11 +56,7 @@ const FormEntry: React.FC<FormEntryComponentProps> = ({ patientUuid, closeWorksp
     }
   }, [state]);
 
-  return (
-    <div>
-      {showForm && selectedForm && patientUuid && patient && <ExtensionSlot name="form-widget-slot" state={state} />}
-    </div>
-  );
+  return <div>{showForm && patientUuid && patient && <ExtensionSlot name="form-widget-slot" state={state} />}</div>;
```

Comment:
Why remove the `selectedForm` check?

Reply:
Ping @donaldkibet , this is a good question. Otherwise this PR LGTM.

Reply:
Hey @brandones and @vasharma05, i removed the selectedForm state variable because it was storing the RXJS subscription value. We've now replaced it with a formInfo prop, which is passed to the FormEntry component.